test.js

```diff
@@ -0,0 +1,15 @@
+const babel = require('babel-core');
```

Comment:
The recommended testing approach is to use [`babel-plugin-tester`](https://github.com/babel-utils/babel-plugin-tester). Would you mind trying that?